workspace.js

```diff
@@ -104,7 +104,7 @@ WorkspaceOverride = {
         const cosmicDock = Main.extensionManager.lookup("cosmic-dock@system76.com");
         translate_x = 0;
 
-        if (cosmicDock && cosmicDock.stateObj) {
+        if (cosmicDock && cosmicDock.stateObj && cosmicDock.stateObj.dockManager.mainDock) {
```

Comment:
Will this cause the fix from https://github.com/pop-os/cosmic-workspaces/pull/36 to not work in this case?

Reply:
Aw man. Good catch, I haven't tested it yet, but you are almost certainly correct.

Do you think this translation logic should get moved out of the init and into the render loop? Maybe [here](https://github.com/pop-os/cosmic-workspaces/blob/master_impish/workspacesView.js#L104)?

Reply:
Aw man. Yep you are correct. I fixed the commit to make the translation in the render loop rather than the init